examples/auth-hook.py

```diff
@@ -0,0 +1,33 @@
+# Example how to roll your own Authorization in the dredd hooks
+import json
+import dredd_hooks as hooks
+import requests
```

Comment:
I'd probably add a comment mentioning this is a third party dependency that they'd need to install.